randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -275,17 +275,21 @@ Extra - asset_id: 6
   * Layers: default
   * Normal Door to Electric Death Room/Door to East Ocean
   > Door to Forgotten Highway Kago Room
-      All of the following:
-          Gravity Suit
-          Hi-Jump Boots or Space Jump
+      Any of the following:
+          All of the following:
+              Gravity Suit
+              Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+          Hi-Jump Boots and Morph Ball and Spring Ball
 
 > Door to Forgotten Highway Kago Room; Heals? False
   * Layers: default
   * Normal Door to Forgotten Highway Kago Room/Door to East Ocean
   > Door to Electric Death Room
-      All of the following:
+      Any of the following:
           Gravity Suit
-          Hi-Jump Boots or Space Jump
+          Hi-Jump Boots and Wall Jumps (Beginner)
+          Space Jump and Wall Jumps (Intermediate)
```

Comment:
What are these two WJs for? (Man, I wish the wiki would have actual content for east ocean...)

Reply:
Jumping out at the far left is not trivial?

Reply:
The water physics makes you need just a little extra jump after you breach the surface

Reply:
Interesting, would've always assumed that you could be able to get out there in vanilla in case you tried to explore without having gravity suit collected. but in vanilla HJ is also required unless you use WJs to skip, which would be the same situation as hre.